src/ServiceControl.Audit/Infrastructure/Bootstrapper.cs

```diff
@@ -168,15 +170,48 @@ long DataSize()
             }
         }
 
+        long FolderSize()
+        {
+            try
+            {
+                var dir = new DirectoryInfo(settings.DbPath);
+                var dirSize = DirSize(dir);
+                return dirSize;
+            }
+            catch
+            {
+                return -1;
```

Comment:
What happens when we return `-1` and pass it through humanizer?

Reply:
@danielmarbach Good question :-)

Reply:
@danielmarbach `-1 B` :-)

Reply:
Seems to work https://dotnetfiddle.net/hJZtd7